llvm/test/MachineVerifier/test_g_vscale.mir

```diff
@@ -0,0 +1,15 @@
+# RUN: not --crash llc -verify-machineinstrs -run-pass none -o /dev/null %s 2>&1 | FileCheck %s
+
+---
+name:            g_vscale
+body: |
+  bb.0:
+
+  %1:_(s32) = G_CONSTANT 4
```

Comment:
This one should also be erroring

Reply:
Ahh, I see. Thanks for the help. I updated the test case.